contracts/TwabRewards.sol

```diff
@@ -157,8 +160,9 @@ contract TwabRewards is ITwabRewards {
         override
         returns (bool)
     {
-        Promotion memory _promotion = _getPromotion(_promotionId);
+        _requireNumberOfEpochs(_numberOfEpochs);
```

Comment:
This isn't quite correct here.  Need to check that the passed _numberOfEpochs is less than (256 - currentNumberOfEpochs)

Reply:
It should be enforced by the `uint8` type no?
It should overflow on the following line: https://github.com/pooltogether/v4-periphery/blob/22094acf51b7645c78c668bab0c2059f045a61f6/contracts/TwabRewards.sol#L168

Reply:
Relying on overflow works, but isn't the best for the user.  They won't know what broke!

Reply:
Done: https://github.com/pooltogether/v4-periphery/pull/34/commits/1f6a4064cce58d4bf6ee7dd8bfa47698d2ce12e1